tests/test_exam_statistics.py

```diff
@@ -0,0 +1,67 @@
+import sys
+
+import pytest
+
+from codecademy.exam_statistics import grades_sum
+from codecademy.exam_statistics import grades_average
+from codecademy.exam_statistics import grades_variance
```

Comment:
You can chain-import from same package (see my comment on your previous PR for details)